README.md

```diff
@@ -55,103 +57,103 @@ Breaking changes:
-| reaction-plugin   | Send a greeting/reaction message when a member joins/leaves/kicked from the guild | Quality of Life | N/A                                                                                                                                |
-| dark-auction      | Send a reminder when a Skyblock Dark Auction is starting                          | Quality of Life | Sends a reminder 5 minutes and 1 minute before the Dark Auction.                                                                   |
-| starfall-cult     | Send a reminder when Skyblock Starfall Cult gathers                               | Quality of Life | N/A                                                                                                                                |
+| Plugin            | Description                                                                             | Importance      | Notice                                                                                                                                |
+| ----------------- | --------------------------------------------------------------------------------------- | --------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| auto-restart      | Schedule restarting every 24 hours                                                      | Essential       | A process monitor is required to restart the application after it shuts down. Not periodically restarting can lead to high RAM usage. |
+| limbo-plugin      | Forces Minecraft client to stay in Hypixel limbo                                        | Essential       | Being outside of limbo for prolonged periods can lead to the account being banned. Only disable if required.                          |
+| punishment-plugin | Enforces punishments and prevents punishment evasion                                    | Recommended     | Plugin will mute users but won't ban them, instead it will send a notification in officer chat.                                       |
+| reaction-plugin   | Send a greeting/reaction message when a member joins/leaves or is kicked from the guild | Quality of Life | N/A                                                                                                                                   |
+| dark-auction      | Send a reminder when a skyblock dark auction is starting                                | Quality of Life | Sends a reminder 5 minutes and 1 minute before the dark auction.                                                                      |
+| starfall-cult     | Send a reminder when the skyblock starfall cult gathers                                 | Quality of Life | N/A                                                                                                                                   |
 
-### How to execute commands
+### How to execute commands (override)
```

Comment:
Changing title will break table of contents links. Need to fix them.

Reply:
I believe ToC should be updated then, title is ambiguous as it only covers one command (`override`) but the title infers all commands

Reply:
Then updated it :)

Reply:
See commit `1cbeeb9a2a4b6cd9c3d05c0d52c4da14e61eb2e5`